src/models/Leaf.ts

```diff
@@ -0,0 +1,7 @@
+export interface ILeaf {
```

Comment:
leaf is just a pair of k->v. so maybe better name will be `IBlockLeafWithProof`?

Reply:
Agree :smile: